src/clr-angular/data/datagrid/animation-hack/row-expand-animation.ts

```diff
@@ -70,6 +70,10 @@ export class DatagridRowExpandAnimation {
   }
 
   private run() {
+    // defense against race condition when rapid, successive toggling occurs and `oldHeight` property is removed
+    if (!this.hasOwnProperty('oldHeight')) {
```

Comment:
I'm wondering why we don't just move `delete this.oldHeight` inside of the `onfinish` handler instead of this guard? I'm worried about side effects. When I run this on 6x slowdown, the animations are janky and it does seem to resolve, but I'd rather delete the oldHeight in the correct place after the animation is done instead of this extra check, as I see the same result.